src/price/ProgressivePrice.php

```diff
@@ -105,8 +105,8 @@ public function calculateSum(QuantityInterface $quantity): ?Money
             $price = $threshold->price();
 
             $chargedAmount = $price->money()
-                                   ->multiply((string)$billedUsage->getQuantity())
-                                   ->divide((string)($price->multiplier()));
+                                   ->multiply((string)(sprintf('%.14F', $billedUsage->getQuantity())))
+                                   ->divide((string)(sprintf('%.14F', $price->multiplier())));
```

Comment:
Please, cover this case with a unit test in https://github.com/hiqdev/php-billing/blob/master/tests/unit/price/ProgressivePriceTest.php